crates/astria-core/src/protocol/transaction/v1/action/group/mod.rs

```diff
@@ -55,6 +61,12 @@ impl_belong_to_group!(
     (FeeAssetChange, Group::BundleableSudo),
     (IbcRelay, Group::BundleableGeneral),
     (IbcSudoChange, Group::UnbundleableSudo),
+    (UpsertMarkets, Group::BundleableGeneral),
+    (CreateMarkets, Group::BundleableGeneral),
+    (UpdateMarkets, Group::BundleableGeneral),
+    (UpdateParams, Group::BundleableSudo),
+    (RemoveMarketAuthorities, Group::BundleableGeneral),
+    (RemoveMarkets, Group::BundleableGeneral),
```

Comment:
Are these groups correct?

Reply:
I think these can all be `BundleableGeneral` for now, as they're not sent by the sudo address